CovidTracker/Pages/Index.cshtml

```diff
@@ -18,11 +18,11 @@
         </div>
         <div>
             <p class="lead">
-                Be <span style="color: orangered"><strong>INFORMED.</strong></span>&nbsp;&nbsp;&nbsp;
-                Be <span style="color: orangered"><strong>PREPARED.</strong></span>&nbsp;&nbsp;&nbsp;
-                Be <span style="color: orangered"><strong>SMART.</strong></span>&nbsp;&nbsp;&nbsp;
-                Be <span style="color: orangered"><strong>SAFE.</strong></span>&nbsp;&nbsp;&nbsp;<br />
-                Be <span style="color: orangered"><strong>READY</strong></span> to fight <span style="color: orangered"><strong>#COVID19</strong></span>
+                Be <span class="highlighted"><strong>INFORMED.</strong></span>&nbsp;&nbsp;&nbsp;
+                Be <span class="highlighted"><strong>PREPARED.</strong></span>&nbsp;&nbsp;&nbsp;
+                Be <span class="highlighted"><strong>SMART.</strong></span>&nbsp;&nbsp;&nbsp;
+                Be <span class="highlighted"><strong>SAFE.</strong></span>&nbsp;&nbsp;&nbsp;<br />
+                Be <span class="highlighted"><strong>READY</strong></span> to fight <span class="highlighted"><strong>#COVID19</strong></span>
```

Comment:
Good.  Linked styles better than inline.  Inline is not permitted with XHTML 1.0 Strict.